contrib/labstack/echo.v4/echotrace.go

```diff
@@ -0,0 +1,55 @@
+// Package echo provides functions to trace the labstack/echo package (https://github.com/labstack/echo).
+package echo
+
+import (
+	"strconv"
+
+	"github.com/labstack/echo/v4"
+
+	"github.com/signalfx/signalfx-go-tracing/ddtrace"
+	"github.com/signalfx/signalfx-go-tracing/ddtrace/ext"
+	"github.com/signalfx/signalfx-go-tracing/ddtrace/tracer"
+	"github.com/signalfx/signalfx-go-tracing/internal/utils"
+)
+
+// Middleware returns echo middleware which will trace incoming requests.
+func Middleware(opts ...Option) echo.MiddlewareFunc {
+	return func(next echo.HandlerFunc) echo.HandlerFunc {
+		cfg := new(config)
+		defaults(cfg)
+		for _, fn := range opts {
+			fn(cfg)
+		}
+		return func(c echo.Context) error {
+			request := c.Request()
+			operationName := c.Path()
```

Comment:
not familiar with echo: is this high-cardinality?

Reply:
I see the tests high-cardinality stuff should be parameters.

Reply:
It might be. I'm not 100% sure but in any case it is preserving the same behavior as we have for pre-v4 so I'm inclined to leave it as is.